src/tests/epicTargetingTestData.ts

```diff
@@ -0,0 +1,166 @@
+    ...SHARED_PARAGRAPHS,
+];
+
+const UK_AUS_LESS_ENGAGED_PARAGRAPHS = [
+    '... we have a small favour to ask. Through these turbulent and challenging times, millions rely on the Guardian for independent journalism that stands for truth and integrity. Readers chose to support us financially more than 1.5 million times in 2020, joining existing supporters in 180 countries.',
+    ...SHARED_PARAGRAPHS,
+];
+
+const US_HIGHLY_ENGAGED_PARAGRAPHS = [
+    '... we have a small favour to ask.  You’ve read %%ARTICLE_COUNT%% articles in the last year. And you’re not alone; through these turbulent and challenging times, millions rely on the Guardian for independent journalism that stands for truth and integrity. Readers chose to support us financially more than 1.5 million times in 2020, joining existing supporters in 180 countries.',
+    ...SHARED_PARAGRAPHS,
+];
+
+const US_LESS_ENGAGED_PARAGRAPHS = [
+    '... we have a small favour to ask. Across the US and around the world, millions rely on the Guardian for independent journalism that stands for truth and integrity. The Guardian has no shareholders or billionaire owner to please, and we invest every penny we earn back into our journalism. Readers chose to support us financially more than 1.5 million times in 2020, joining existing supporters in 180 countries.',
+    ...SHARED_PARAGRAPHS,
+];
+
+const EU_ROW_HIGHLY_ENGAGED_PARAGRAPHS = [
+    '… as you join us today from %%COUNTRY_NAME%%, we have a small favour to ask. You’ve read %%ARTICLE_COUNT%% articles in the last year. And you’re not alone; through these turbulent and challenging times, millions rely on the Guardian for independent journalism that stands for truth and integrity. Readers chose to support us financially more than 1.5 million times in 2020, joining existing supporters in 180 countries.',
+    ...SHARED_PARAGRAPHS,
+];
+
+const EU_ROW_LESS_ENGAGED_PARAGRAPHS = [
+    '... as you join us today from %%COUNTRY_NAME%%, we have a small favour to ask. Through these turbulent and challenging times, millions rely on the Guardian for independent journalism that stands for truth and integrity. Readers chose to support us financially more than 1.5 million times in 2020, joining existing supporters in 180 countries.',
+    ...SHARED_PARAGRAPHS,
+];
+
+// ---- Other variant fields ---- //
+
+const HIGHLIGHTED_TEXT =
+    'Support the Guardian from as little as %%CURRENCY_SYMBOL%%1 – it only takes a minute. If you can, please consider supporting us with a regular amount each month. Thank you.';
+
+const CTA: Cta = {
+    text: 'Support The Guardian',
+    baseUrl: 'https://support.theguardian.com/contribute',
+};
+
+// ---- Variants ---- //
+
+const BASE_VARIANT: Variant = {
```

Comment:
very minor thing but if you remove the `Variant` type here and leave out `name` and `paragraphs` then TS should typecheck its uses below

Reply:
That's a great shout - I can do the same with the `BASE_TEST` too

Reply:
Actually because of the `userCorhort` type I think it's easier to not change that one